pkg/codegen/testing/test/testdata/other.json

```diff
@@ -16,6 +16,13 @@
           "type": "number"
         }
       }
+    }, 
+    "pulumi:providers:other": {
+      "inputProperties": {
+        "name": {
+          "type": "string"
+        }
+      }
```

Comment:
There is a `"provider"` section of the JSON. Providers are not standard resources.